draft-ietf-quic-multipath.md

```diff
@@ -1246,8 +1251,10 @@ also needs to be considered in the context of the Path Identifier field.
 A MAX_PATH_ID frame (type=0x15228c0c) informs the peer of the maximum path identifier
 it is permitted to use.
 
-When there are not enough unused path identifiers, endpoints SHOULD
-send MAX_PATH_ID frame to inform the peer that new path identifiers are available.
+When there are not enough unused path identifiers, endpoints SHOULD either send a
+MAX_PATH_ID frame to increase the active path limit (when limited by the sender)
+or a MAX_PATH_ID_BLOCKED frame to inform the peer that a new path identifier was needed
+but the limit of active paths set by the peer has been reached.
```

Comment:
PR #386 removes this sentence here as we should only specify it once in the doc.